### PR TITLE
DOCS-5645: Remove private beta from DBM docs

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do?
Removes private beta from Oracle DBM docs.

### Motivation
DOCS-5645

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: config/_default/menus/menus.en.yaml [hunk 1]
```diff
@@ -2413,36 +2413,71 @@ main:
     parent: dbm_setup_sql_server
     identifier: dbm_troubleshooting_sql_server
     weight: 107
+  - name: Setting up Oracle
```
Comment (caseycull) [inline config/_default/menus/menus.en.yaml:2416]: I noticed we capitalize the u in "up" here for the child pages, e.g. https://docs-staging.datadoghq.com/deforest/dbm-oracle-beta/database_monitoring/setup_oracle/selfhosted/

File: content/en/database_monitoring/setup_oracle/_index.md [hunk 2]
```diff
@@ -3,22 +3,20 @@ title: Setting up Oracle
...
 <div class="alert alert-info">
-The features described on this page are in private beta.
+The features described on this page are in beta. Contact your Customer Success Manager to provide feedback or ask for help.
 </div>
 
 ## Supported Oracle versions, features, and architectures
 
 - **Versions**: 19c and 21c
-- **Deployment configurations**: Self-managed, RDS, RAC, Exadata
+- **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous database
```
Comment (caseycull) [inline content/en/database_monitoring/setup_oracle/_index.md:19]: Should the d in "database" be capitalized?

File: config/_default/menus/menus.en.yaml (hunk 1 above)
Comment (drichards-87) [reply config/_default/menus/menus.en.yaml:2416]: Yes! Good catch. I'll fix it.

Comment (drichards-87) [inline config/_default/menus/menus.en.yaml:2416]:
```suggestion
  - name: Setting Up Oracle
```

File: content/en/database_monitoring/setup_oracle/_index.md (hunk 2 above)
Comment (drichards-87) [reply content/en/database_monitoring/setup_oracle/_index.md:19]: I'll update this one as well.

Comment (drichards-87) [inline content/en/database_monitoring/setup_oracle/_index.md:19]:
```suggestion
- **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous Database
```

**Review by caseycull — APPROVED** [review]

**Review by jhgilbert — APPROVED** [review]

**Pull request merged**